ui/src/app/components/runtime-configuration-panel.tsx

```diff
@@ -0,0 +1,1079 @@
+import * as React from 'react';
```

Comment:
moved ~2500 lines of `pages/analysis/runtime-panel` to:
* ~1000 lines to this file `components/runtime-configuration-panel.tsx`
* ~1500 lines to **components in the dir** `components/runtime-configuration-panel`